src/libs/actions/Policy.js

```diff
@@ -718,7 +719,23 @@ function generateDefaultWorkspaceName(email, policyList) {
         defaultWorkspaceName = 'My Group Workspace';
     }
 
-    return hasPolicies ? `${defaultWorkspaceName} ${policyListLength + 1}` : defaultWorkspaceName;
+    if (defaultWorkspaceName === '') {
+        return defaultWorkspaceName;
+    }
+    console.log('this is all the polices ', getAllPolicies());
+
+    // Check if this name already exists in the policies
+    let suffix = 0;
+    _.forEach(getAllPolicies(), (policy) => {
```

Comment:
Should be able to just use allPolicies here instead of this method - were you trying to mock it?